salt/states/boto3_route53.py

```diff
@@ -798,3 +826,204 @@ def rr_absent(name, HostedZoneId=None, DomainName=None, PrivateZone=False,
         ret['result'] = False
 
     return ret
+
+def health_check_present(name, Name=None, CallerReference=None, IPAddress=None,
+                         Port=None, Type=None, ResourcePath=None, FullyQualifiedDomainName=None, SearchString=None,
+                         RequestInterval=None, FailureThreshold=None, MeasureLatency=None, Inverted=None, Disabled=None,
+                         HealthThreshold=None, ChildHealthChecks=None, EnableSNI=None, Regions=None, AlarmIdentifier=None,
+                         InsufficientDataHealthStatus=None, region=None, key=None, keyid=None, profile=None):
+    '''
+    Ensure the Route53 health check is present.
+
+    name
+        The name of the state definition.  This will be used for Name if the latter is
+        not provided.
+
+    Name
+        Name of the health check.
+
+    region
+        The region to connect to.
+
+    key
+        Secret key to be used.
+
+    keyid
+        Access key to be used.
+
+    profile
+        Dict, or pillar key pointing to a dict, containing AWS region/key/keyid.
+    '''
+
+    Name = Name if Name else name
```

Comment:
```suggestion
Name = Name or name
```

Though... is it necessary to have both `Name` *and* `name`? I'm pretty sure that other states simply do something like:

```
foo:
   test.nop:
    - name: This is my name now!
```

That seems like it would make this function a bit easier to follow.

Reply:
This pattern was taken from boto_cloudtrail & boto_cloudwatch_event.